zulipterminal/helper.py

```diff
@@ -92,29 +92,30 @@ def _set_count_in_model(new_count: int, changed_messages: List[Message],
         but updates `unread_counts` (which can update the model
         if it's passed in, but is not tied to it).
     """
+    # broader unread counts (for all_* and streams) are updated
+    # later conditionally.
```

Comment:
In the previous refactoring we missed the point that this now occurs 'later' - in another method! So it would be useful to improve the comment here.